pkgs/top-level/python-aliases.nix

```diff
@@ -79,6 +79,10 @@ mapAliases ({
   qasm2image = throw "qasm2image is no longer maintained (since November 2018), and is not compatible with the latest pythonPackages.qiskit versions."; # added 2020-12-09
   requests_toolbelt = requests-toolbelt; # added 2017-09-26
   rotate-backups = throw "rotate-backups was removed in favor of the top-level rotate-backups"; # added 2021-07-01
+  ruamel_base = ruamel-base; # added 2021-11-01
+  ruamel_ordereddict = ruamel-ordereddict; # added 2021-11-01
```

Comment:
This seems to only exist in the python2 package set.

Reply:
It does. I am not sure wether we should add an alias for it but if it causes an eval error the answer is no.